src/components/pages/Manage/Manage.tsx

```diff
@@ -2,64 +2,137 @@ import { useEffect, useState } from 'react';
 
 import { useWalletAddress } from '../../../hooks';
 import { useGlobalState } from '../../../state/contexts/GlobalState';
-import { CopyIcon } from '../../icons';
+import { CodeSandboxIcon, NotebookIcon } from '../../icons';
+import {
+  CalendarTimeIcon,
+  LinkIcon,
+  PriceTagIcon,
+  RefreshAlertIcon,
+} from '../../icons';
 import { Loader } from '../../layout';
+import { AntTable, NameTable } from '../../layout/tables';
+import './styles.css';
 
 function Manage() {
   const [{ arnsSourceContract }] = useGlobalState();
   const { walletAddress, wallet } = useWalletAddress();
+
+  const [tableType, setTableType] = useState('ant'); // ant or name
```

Comment:
use an enum instead of just a string.


i.e. 
```
export enum TABLE_TYPES {
    ANT: 'ant_table'
    NAME: 'name_table'
}
```
and reference it via `TABLE_TYPE.ANT`

Reply:
added